### ISSUE TITLE
Support ATT&CK v14.1 Techniques

### ISSUE THREAD

**Issue by mr-tz** [fireeye/capa-rules#887]
Thank you very much for your contributions.
May I ask if you have any plans in the near future to synchronize capa rules with ATT&CK v14.1?

_Originally posted by @Iroxious in https://github.com/mandiant/capa-rules/discussions/222#discussioncomment-8661492_

I see the following potential new techniques:
- [ ] Exfiltration Over Web Service: Exfiltration Over Webhook
- [ ] Hide Artifacts: Ignore Process Interrupts
- [ ] Impair Defenses: Disable or Modify Linux Audit System
- [ ] Log Enumeration
- [ ] Masquerading: Break Process Trees 
- [ ] Power Settings 
- [ ] System Network Configuration Discovery: Wi-Fi Discovery

**Comment by Iroxious** [comment]
Thank you for your response. May I understand that for other earlier techniques not mentioned by you, if they can be utilized in CAPA analysis, they are already roughly included in capa-rules? (I saw that capa-rules currently involve a total of 120 TTP techniques.)

**Comment by mr-tz** [comment]
Yes, we don't exactly track the supported version or coverage but many rules have ATT&CK and MBC mappings included.

**Comment by Iroxious** [comment]
I understand, thank you for your explanation. :)
